right/src/utils.c

```diff
@@ -176,3 +176,13 @@ bool ShouldResendReport(bool statusOk, uint8_t* counter) {
         return false;
     }
 }
+
+uint8_t IsRightHalfOrModule(uint16_t id)
```

Comment:
```suggestion
static uint8_t isRightHalfOrModule(uint16_t id)
```

If a function is not used from another compilation unit, we prefer it to be static and dromedaryCase. (Allows compiler to inline it, and makes clear what is actually used project-wide and what not.)

Reply:
I remember reading that, actually.  Woopsie